### PR TITLE
Plot velocity bounds on top of spectrums

### PR DESCRIPTION
**Pull request by alexanderkuvaev**
* If a stacking velocity is displayed on top of the vertical velocity spectrum, the bounds it was calculated in can be highlighted if `plot_bounds=True` (default)
* Similarly, an area around central stacking velocity can be highlighted on the residual spectrum plot if its size is passed as `acceptable_margin`. It may be used for visual quality control of stacking velocity picking by setting this value low enough (e.g. 2%) and checking that local maximas of velocity spectrum corresponding to primaries lie inside the highlighted area.
* Some new properties were added to spectrum classes
* And some visualization bugs were fixed

spectrum

### CODE REVIEW

File: seismicpro/velocity_spectrum/velocity_spectrum.py [hunk 1]
```diff
@@ -418,40 +430,49 @@ def get_velocity_range(self, stacking_velocity, relative_margin, velocity_step):
...
+            time = self.delay + self.sample_interval * time_ix
 
-        if (velocity_ix < 0) or (velocity_ix >= len(self.velocities)):
-            velocity = None
-        else:
-            velocity = np.interp(velocity_ix, np.arange(len(self.velocities)), self.velocities)
-            velocity /= 1000  # from m/s to m/ms
+        velocity = None
+        if 0 <= velocity_ix <= self.n_velocities - 1:
+            velocity = np.interp(velocity_ix, np.arange(self.n_velocities), self.velocities) / 1000  # from m/s to m/ms
```
Comment (nikita-klsh) [inline seismicpro/velocity_spectrum/velocity_spectrum.py:439]: please unify units in comment with units in docs

File: seismicpro/velocity_spectrum/velocity_spectrum.py [hunk 2]
```diff
@@ -625,14 +649,18 @@ class ResidualVelocitySpectrum(BaseVelocitySpectrum):
...
         Stacking velocity around which residual velocity spectrum was calculated.
-    relative_margin : float, optional, defaults to 0.2
-        Relative velocity margin, that determines the velocity range for velocity spectrum calculation for each time.
     velocity_spectrum : 2d np.ndarray
         An array with calculated residual vertical velocity spectrum values.
+    margins : 1d np.ndarray
+        An array of velocity margins the spectrum was calculated for.
+    margin_step : float
+        A step between each two adjacent margins.
+    relative_margin : float, optional, defaults to 0.2
```
Comment (nikita-klsh) [inline seismicpro/velocity_spectrum/velocity_spectrum.py:660]:
```suggestion
    relative_margin : float
```

**Review by nikita-klsh — APPROVED** [review]

File: seismicpro/velocity_spectrum/velocity_spectrum.py (hunk 2 above)
Comment (alexanderkuvaev) [reply seismicpro/velocity_spectrum/velocity_spectrum.py]: Fixed

File: seismicpro/velocity_spectrum/velocity_spectrum.py (hunk 1 above)
Comment (alexanderkuvaev) [reply seismicpro/velocity_spectrum/velocity_spectrum.py:439]: Fixed in all three methods

**Review by anton-br — APPROVED** [review]

**Pull request merged**